business/checkers/virtual_services/no_gateway_checker_test.go

```diff
@@ -19,7 +19,7 @@ func TestMissingGateway(t *testing.T) {
 	virtualService := data.AddGatewaysToVirtualService([]string{"my-gateway", "mesh"}, data.CreateVirtualService())
 	checker := NoGatewayChecker{
 		VirtualService: virtualService,
-		GatewayNames:   make(map[string]struct{}, 0),
+		GatewayNames:   make(map[string]struct{}),
```

Comment:
This isn't functionally equivalent to the ``, 0`` version. This one allocates "enough memory" (undefined).

Reply:
it doesn't seem so : https://staticcheck.io/docs/checks#S1019